configs/ASRock/A300M-STX-DeskMini.conf

```diff
@@ -0,0 +1,95 @@
+# ASRock A300M-STX (as included in DeskMini A300 barebone)
+# manufacturing date April 2020 (came with UEFI BIOS 3.60)
+# board does have connectors for 2 fans (CPU_FAN1 and CPU_FAN2)
+# board does have a connector for intrusion detection
+# board does not have a speaker connector
+# dmesg: "nct6775: Found NCT6793D or compatible chip at 0x2e:0x290"
+
+chip "nct6793-*"
+
+   # board has no speaker connector
+   set beep_enable 0
+
+   # voltages
+   label   in0     "Vcore"
+   compute in0     @*2, @/2
+   set     in0_min 0.16
+   set     in0_max 1.46
+
+   label   in2     "AVCC"
+   set     in2_min 3.30 * 0.90
+   set     in2_max 3.30 * 1.10
+
+   label   in3     "+3.3V"
+   set     in3_min 3.30 * 0.90
+   set     in3_max 3.30 * 1.10
+
+   label   in7     "3VSB"
+   set     in7_min 3.30 * 0.90
+   set     in7_max 3.30 * 1.10
+
+   label   in8 "VBAT"
+   set     in8_min 3.00 * 0.90
+   set     in8_max 3.30 * 1.10
+
+   label   in9 "+12V"
+   compute in9 @*(53/8), @/(53/8)
+   set     in9_min 12 * 0.90
+   set     in9_max 12 * 1.10
+
+   label   in13     "+5V"
+   compute in13 @*(24/8), @/(24/8)
+   set     in13_min 5 * 0.90
+   set     in13_max 5 * 1.10
+
+   # these have non-zero input, but are unknown
+   ignore  in1
+   ignore  in4
+   ignore  in5
+   ignore  in6
+   ignore  in10
+   ignore  in11
+   ignore  in12
+   ignore  in14
+
+   # temperatures
+   # not sure about temp2 and temp3
+   label   temp2 "VRM(?) temp"
+   set temp2_max      75
+   set temp2_max_hyst 70
+
+   label   temp3 "MB(?) temp"
+   set temp3_max      55
+   set temp3_max_hyst 50
+
+   label   temp7 "CPU temp"
+
+   # other temps are zero or unreasonably high/low
+   ignore  temp1
+   ignore  temp4
+   ignore  temp5
+   ignore  temp6
+   ignore  temp8
+   ignore  temp9
+   ignore  temp10
+
+   # fans
+   label fan2 "CPU_FAN1"
+   set fan2_min 500
+
+   label fan1 "CPU_FAN2"
+   set fan1_min 500
+   # it's near impossible to fit a 2nd fan (besides the CPU fan)
+   # in the DeskMini case, so ignore unconnected CPU_FAN2
+   ignore fan1
```

Comment:
That's funny :). Even in this case, I think I'd prefer not to ignore the fan in the upstream config.

Reply:
agree, changed